lib/ioki/apis/endpoints/create.rb

```diff
@@ -40,7 +40,7 @@ def call(client, model, args = [], options = {})
           params: options[:params]
         )
 
-        return if parsed_response.nil?
+        return if Support.blank?(parsed_response)
```

Comment:
When a response is an empty string, it will not be cast to `nil` anymore, so we check for `empty?` or `nil?`